.github/workflows/pull-request.yml

```diff
@@ -3,19 +3,31 @@ name: Pull Request
 on: [ pull_request ]
 
 jobs:
+  nix_build:
+    runs-on: ubuntu-20.04
```

Comment:
any reason why there's an explicit ubuntu version here for PRs, but the build & push workflow uses `ubuntu-latest`?